src/windows/staff.c

```diff
@@ -1325,6 +1325,17 @@ void window_staff_viewport_init(rct_window* w){
 	window_invalidate(w);
 }
 
+void staff_get_valid_skins(int* ebx)
+{
+	init_scenery();
+	*ebx = 0;
```

Comment:
Using your comments, I edited the function(s). Any further ideas for improvement?